soc/arm/nordic_nrf/nrf91/mpu_regions.c

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2017 Linaro Limited.
+ * Copyright (c) 2018 Nordic Semiconductor ASA.
+ *
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <misc/slist.h>
+#include <arch/arm/cortex_m/mpu/arm_mpu.h>
+
+
+#define XICR_BASE	0x10000000
```

Comment:
Do we have any header with these value defined?
Why we are not fetching these addresses from dts?
Why we have defines which are not used?

Reply:
I think this can be cleaned-up. These defines are in the .c file, thus, cannot be used outside this source file. I added them to be consistent with nRF52 but I see that they are neither used in nRF52. Similar defines are added in some NXP and ST SOCs. I suggest open a PR to clean them up, since they are not used.

Reply:
Removed, already, from this file.